### PR TITLE
Optimize `SubscribeCommitteeSubnets` VC action

### PR DESCRIPTION
**Pull request by rkapka**
**What type of PR is this?**

Other

**What does this PR do? Why is it needed?**

`SubscribeCommitteeSubnets` in the REST VC takes around 1-2 seconds. Almost all of this time is spend on fetching duties to get the number of committees in a slot, but we already return this value when fetching duties at the beginning of the epoch, we just need to pass it into the function. Adding `committees_at_slot` to the duty proto message allows us to have this value available.

This reduces the time needed to subscribe from 1-2 seconds to 1 millisecond.

### CODE REVIEW

File: proto/prysm/v1alpha1/beacon_block.pb.go [hunk 1]
```diff
@@ -3899,7 +3899,7 @@ var file_proto_prysm_v1alpha1_beacon_block_proto_rawDesc = []byte{
 	0x64, 0x72, 0x61, 0x77, 0x61, 0x6c, 0x73, 0x2e, 0x70, 0x72, 0x6f, 0x74, 0x6f, 0x1a, 0x26, 0x70,
 	0x72, 0x6f, 0x74, 0x6f, 0x2f, 0x65, 0x6e, 0x67, 0x69, 0x6e, 0x65, 0x2f, 0x76, 0x31, 0x2f, 0x65,
 	0x78, 0x65, 0x63, 0x75, 0x74, 0x69, 0x6f, 0x6e, 0x5f, 0x65, 0x6e, 0x67, 0x69, 0x6e, 0x65, 0x2e,
-	0x70, 0x72, 0x6f, 0x74, 0x6f, 0x22, 0xf4, 0x05, 0x0a, 0x18, 0x47, 0x65, 0x6e, 0x65, 0x72, 0x69,
+	0x70, 0x72, 0x6f, 0x74, 0x6f, 0x22, 0xee, 0x05, 0x0a, 0x18, 0x47, 0x65, 0x6e, 0x65, 0x72, 0x69,
```
Comment (james-prysm) [inline proto/prysm/v1alpha1/beacon_block.pb.go:3902]: Why was this file changed if no proto changed?

File: validator/client/beacon-api/duties.go [hunk 2]
```diff
@@ -195,14 +200,15 @@ func (c beaconApiValidatorClient) getDutiesForEpoch(
...
-			IsSyncCommittee: syncDutiesMapping[validatorIndex],
+			Committee:        committeeValidatorIndices,
+			CommitteeIndex:   committeeIndex,
+			AttesterSlot:     attesterSlot,
+			ProposerSlots:    proposerDutySlots[validatorIndex],
+			PublicKey:        pubkey,
+			Status:           validatorStatus.Status,
+			ValidatorIndex:   validatorIndex,
+			IsSyncCommittee:  syncDutiesMapping[validatorIndex],
+			CommitteesAtSlot: slotCommittees[strconv.FormatUint(uint64(attesterSlot), 10)],
```
[the remark this answers is not in the document]
Comment (rkapka) [reply validator/client/beacon-api/duties.go:211]: Because that would lead to more code. Inside the committees `for` loop (that you commented on above) I would have to convert `c.Slot` to a number at each iteration, which includes an error check.

**Review by james-prysm — APPROVED** [review]
